stories/measurementInfoPanel.stories.tsx

```diff
@@ -1,11 +1,12 @@
 import { Meta } from '@storybook/react';
-import { useState, useEffect } from 'react';
+import axios from 'axios';
+import { QueryClient, QueryClientProvider, useQuery } from 'react-query';
```

Comment:
In the issue we were thinking about something like `useMeasurementExample` and the currently implementation seems qui complex to me especially that this load of the example will be reused in many places.

@targos any advice to simplify this code ? Loading the example data should be only one line in a story book I think.

Reply:
Yes we need to use a global decorator to wrap all stories with the query provider

Reply:
> Yes we need to use a global decorator to wrap all stories with the query provider

I tired it and it cause a bug, because when we use a global decorator it will be the same QueryClient so when we change story data will stay the same as the previous one

Reply:
I used the decoration now